tests/kubernetes/kubernetes_cluster_connector_test.py

```diff
@@ -50,53 +102,9 @@ def mock_cluster_connector():
         }
         mock_cluster_connector._pods_by_ip = {
             '10.10.10.1': [],
-            '10.10.10.2': [
-                V1Pod(
-                    metadata=V1ObjectMeta(name='pod1'),
-                    status=V1PodStatus(phase='Running'),
-                    spec=V1PodSpec(containers=[
-                           V1Container(
-                                name='container1',
-                                resources=V1ResourceRequirements(requests={'cpu': '1.5'})
-                            )
-                        ]
-                    )
-                ),
-            ]
+            '10.10.10.2': [pod1, pod2],
```

Comment:
I might suggest having a case where there is a pod that is safe to evict on a node to show that that node is killable since that seems like a major case.